Jenkinsfile

```diff
@@ -6,11 +6,11 @@ node {
   govuk.buildProject(
     rubyLintDiff: false,
     beforeTest: {
-      stage("Install npm dependencies") {
-        sh("npm install")
+      stage("Install yarn dependencies") {
+        sh("yarn")
       }
       stage("Lint Javascript and SCSS") {
-        sh("npm run lint")
+        sh("yarn run lint")
```

Comment:
The linting is also part of the default rake task - does including it in the Jenkinsfile mean that the linting will run twice?

Reply:
Oh good point, I'll see if it can be removed.